style.css

```diff
@@ -257,6 +257,25 @@ hr {
 }
 /* End  */
 
+/* backtotop */
+#goToTopBtn {
+  position: fixed; /* Fixed/sticky position */
+  bottom: 20px; /* Place the button at the bottom of the page */
+  right: 30px; /* Place the button 30px from the right */
+  z-index: 99; /* Make sure it does not overlap */
+
+  border-radius: 25px;

```

Comment:
you may make border-radius 50% to make the button circle.  I thinks it will look better